src/main/java/org/opentripplanner/middleware/models/TripRequest.java

```diff
@@ -82,4 +92,33 @@ public boolean delete() {
         }
         return Persistence.tripRequests.removeById(this.id);
     }
+
+    private AnonymizedTripRequest getAnonimized() {
+        return new AnonymizedTripRequest(batchId, fromPlace, toPlace);
+    }
+
+    /**
+     * Get all trip requests between two dates.
+     */
+    private static FindIterable<TripRequest> getTripRequests(Date start, Date end) {
```

Comment:
It may just be a personal opinion, but since some of these static methods are only ever called once and from the ConnectedDataManager class that perhaps these methods belong in that class instead.